config/webpacker.yml

```diff
@@ -59,6 +59,15 @@ test:
   # Compile test packs to a separate directory
   public_output_path: packs-test
 
+staging:
```

Comment:
Can't we use production environment settings? I thought the buildpack was already precompiling the assets in staging. It's supposed to be 100% similar to production as far as the infrastructure goes.

Reply:
It's the same, you need this though for it to work.

Reply:
Check it out now, I made it inherit from production.

Reply:
Oooh, I like this